pkg/master/master.go

```diff
@@ -47,9 +46,7 @@ import (
 	extensionsapiv1beta1 "k8s.io/kubernetes/pkg/apis/extensions/v1beta1"
 	networkingapiv1 "k8s.io/kubernetes/pkg/apis/networking/v1"
 	policyapiv1beta1 "k8s.io/kubernetes/pkg/apis/policy/v1beta1"
-	rbacv1alpha1 "k8s.io/kubernetes/pkg/apis/rbac/v1alpha1"
```

Comment:
I knew the path was stable, but I thought we had already started cleaning up the shared global `api.Scheme` and you had to opt into the groups registered into your storage scheme